bin/test-image

```diff
@@ -76,7 +76,7 @@ if [ "${TYPE}" = "vm" ]; then
 
     # Some distros don't support secure boot.
     case "${DISTRO}" in
-        alpine|archlinux|gentoo|nixos)
+        alpine|archlinux|gentoo|nixos|openeuler)
```

Comment:
I tried to find if lxc-ci had a better fix but apparently they always disable secure boot in their image tests (https://github.com/lxc/lxc-ci/blob/main/bin/test-image#L48)